server/controllers/apply/withdrawablesController.test.ts

```diff
@@ -41,13 +42,14 @@ describe('withdrawablesController', () => {
         const placementRequestWithdrawable = withdrawableFactory.build({ type: 'placement_request' })
         const placementApplicationWithdrawable = withdrawableFactory.build({ type: 'placement_application' })
         const applicationWithdrawable = withdrawableFactory.build({ type: 'application' })
-        const withdrawables = [placementRequestWithdrawable, placementApplicationWithdrawable, applicationWithdrawable]
+        const withdrawable = [placementRequestWithdrawable, placementApplicationWithdrawable, applicationWithdrawable]
```

Comment:
This should to remain `withdrawbles` given there are multiple withdrawables here